pages/api/users/models/UserSchema.ts

```diff
@@ -1,13 +1,18 @@
 import { Schema, model, models } from 'mongoose';
 import bcrypt from 'bcryptjs';
 import { Gender } from '../../../../types/common.types';
-import { isEmail, isUrl } from 'js-string-helper';
+import { isEmail, isPhoneNumber, isUrl } from 'js-string-helper';
```

Comment:
@JRakin always keep same validation is backend and frontend. phone number validation in schema and formik is not matching. 

Better not to use validation for phone number in our case. we dont need